src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -298,6 +332,79 @@
                     "description": "Recordings of test runs from a single node"
                 }
             }
+        },
+        "raft_metrics": {
+            "id": "raft_metrics",
+            "description": "Raft metrics",
+            "properties": {
+                "leader_info": {
+                    "type": "raft_metrics_leader",
+                    "description": "Leader info"
+                },
+                "followers": {
+                    "type": "array",
+                    "items": {
+                        "type": "raft_metrics_follower"
+                    },
+                    "description": "Raft followers info"
+                }
+            }
+        },
+        "raft_metrics_leader": {
+            "id": "raft_metrics_leader",
+            "description": "Raft metrics leader",
+            "properties": {
+                "id": {
+                    "type": "int",
+                    "description": "Node id"
+                },
+                "committed_log_index": {
+                    "type": "long",
+                    "description": "Commited log index"
+                },
+                "dirty_log_index": {
+                    "type": "long",
+                    "description": "Dirty log index"
+                }
+            }
+        },
+        "raft_metrics_follower": {
```

Comment:
maybe there is a better term to use than `_metrics`? `_info` or `_state` or `_???` might be better since i think metrics has a particular meaning in another context that isn't quite the same. wdyt?

Reply:
state should be better! Thanks